apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -433,14 +456,12 @@ export async function sync(
       z.string(),
     );
     const questionSharingEnabled = await features.enabled('question-sharing', {
-      course_id: courseId,
-      course_instance_id: courseInstanceId,
```

Comment:
Including a course instance ID wasn't necessary here; we have a rule that we'll only ever change these features for either entire institutions or courses, never individual course instances.